internal/vehicle/ford.go

```diff
@@ -134,36 +170,139 @@ func (v *Ford) vehicles() ([]string, error) {
 	return vehicles, err
 }
 
-// chargeState implements the api.Vehicle interface
-func (v *Ford) chargeState() (float64, error) {
-	var resp struct {
-		VehicleStatus struct {
-			Battery struct {
-				BatteryStatusActual struct {
-					Value int
-				}
+// Parses vehicle timestamp in format returned by Ford API and calculates age
+func (v *Ford) CalculateAge(timestamp string) (age time.Duration, err error) {
+	var timestampTime time.Time
+	const dateFormat = "01-02-2006 15:04:05"
+	timestampTime, err = time.Parse(dateFormat, timestamp)
+	age = time.Since(timestampTime)
```

Comment:
Given this function is really only 2 lines of actual code you could probably inline it.

Reply:
it's not inlined because I have two references

Reply:
You can still write that much shorter and make it private